docs/src/components/pricing/PricingTable.tsx

```diff
@@ -1201,105 +1162,105 @@ export default function PricingTable({
           Data Grid
         </Button>
       </Box>
-      <Collapse in={dataGridCollapsed} timeout={700} sx={{ position: 'relative' }}>
-        <Box
-          sx={(theme) => ({
-            position: 'absolute',
-            width: '2px',
-            left: 10,
-            top: 0,
-            bottom: 0,
-            bgcolor: 'grey.100',
-            ...theme.applyDarkStyles({
-              bgcolor: 'primaryDark.700',
-            }),
-          })}
-        />
+      <Collapse
+        in={dataGridCollapsed}
+        timeout={700}
+        sx={(theme) => ({
+          position: 'relative',
+          ml: 1.5,
+          borderLeftWidth: '2px',
+          borderLeftStyle: 'solid',
+          borderColor: 'grey.100',
+          ...theme.applyDarkStyles({
+            borderColor: 'primaryDark.700',
+          }),
+        })}
+      >
```

Comment:
Here is the main modification.

For a reason I ignore, all the nested element were nested thanks to some margin, and the left bar were made thanks to a box with absolute position.

I replaced all that with a margin-left and border-left on the Collapse item, such that all the children are translated to the right. Which allows to remove props `nested` everywhere.

I did not found issue when resizing the window, including with smartphone debugger

Reply:
Bad thing about this approach, it seems, is that the rows inside the accordion are now misaligned with the rest of the columns/rows:

<img width="600" alt="Screen Shot 2023-08-28 at 19 08 53" src="https://github.com/mui/material-ui/assets/67129314/77466b95-3102-48d8-b4b1-332479ec62fb">